source/javascripts/modules/highlighted-nav.js

```diff
@@ -0,0 +1,47 @@
+(function($, Modules) {
+  'use strict';
+
+  Modules.HighlightedNav = function HighlightedNav() {
```

Comment:
It's largely to assist with debugging – a named function like this will appear with that name in the JS console in the event of an error occurring within it, as opposed to `(anonymous function)`. It feels a bit weird to be duplicating it, but can be a bacon-saver in tracking down weird bugs in my experience.